python/ccf/checker.py

```diff
@@ -27,7 +27,7 @@ def wait_for_global_commit(client, seqno, view, mksign=False, timeout=3):
 
     end_time = time.time() + timeout
     while time.time() < end_time:
-        r = client.get("/node/tx", {"view": view, "seqno": seqno})
+        r = client.get(f"/node/tx/{view}/{seqno}")
```

Comment:
I could be swayed otherwise but I find this syntax a little odd. It makes it look as if `view` was a resource and `seqno` another resource under `view`. In practice, it's kind of true (a committed `seqno` belongs to one `view`).

Reply:
A fair observation. I don't have strong opinions either way, I'm happy to switch this back to query params if that's what people prefer. I'm just thinking about what the `/{view}` resource could/would hold, the leader in that term and the begin/end seqnos in that term?

Reply:
Yeah, I'm not too strongly for one rather than the other either. `/view` should probably return the begin/end seqnos as you said. An interesting case to consider is when `seqno` does not belong to `view`: should we return HTTP `404`? I believe we currently return `UNKNOWN` or `COMMITTED` in this case and should continue to do so.

Reply:
Since this isn't really a clear-cut improvement over the current state, I suggest we stay compatible?

Reply:
Agreed, have reverted this to pass in query rather than path.